### ISSUE TITLE
Optimizations to be made

### ISSUE THREAD

**Issue by LasseRafn** [spatie/laravel-activitylog#580]
Would you accept a PR speeding this method up, by removing the Collection class? We're literally instantiating a new Collection only to pick one attribute and converting back to an array, for EVERY attribute to log.

This line:
```php
$changes += collect($model)->only($attribute)->toArray();
```

Can be changed to:
```php
$changes += [$model->{$attribute}];
```

And I'm seeing a crazy improvement in performance (see screenshot on my app saving changes a few times on a single model)

<img width="211" alt="Screenshot 2019-08-10 16 24 26" src="https://user-images.githubusercontent.com/2689341/62823676-66241600-bb8b-11e9-8141-f92836d45a03.png">
<img width="193" alt="Screenshot 2019-08-10 16 24 51" src="https://user-images.githubusercontent.com/2689341/62823677-66241600-bb8b-11e9-87c1-409d2a1b4d06.png">

(do note the ms times are ~5x larger due to Blackfire running..)

**Comment by Gummibeer** [comment]
Could you provide the line you mean and version you are on?
I think you refer to https://github.com/spatie/laravel-activitylog/blob/ef174eaf65f04e64f17cf01d477a48002a3d8c62/src/Traits/DetectsChanges.php#L116-L139
Because this method was refactored in #534 which were released in `v3.6.0`. So I beliefe that you are on an older version?

**Comment by LasseRafn** [comment]
You're absolutely right! My bad! 😅
